trainee_template/explanation_residuals.amlg

```diff
@@ -245,6 +245,25 @@
 			local_null_conviction_ratios_map (null)
 		))
 
+		;if case_residuals_map contains any .nan's, add those features to the inactive_features_zeros_map
+		(if (contains_value case_residuals_map .nan)
```

Comment:
too broad, what if the nan was actually caused by something else? this solution isn't viable.